conf/application.conf

```diff
@@ -54,6 +56,12 @@ microservice {
       host = localhost
       port = 8500
     }
+
+    pds {
+      protocol = http
+      host = localhost
+      port = 10162
```

Comment:
this might be ambiguous - especially since this is the pds-checker. Perhaps pds-auth-checker-api-eis-stub or even just eis-stub could do the trick.